src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.scss

```diff
@@ -0,0 +1,99 @@
+@import '../../../common/variables';
+@import './mixins';
+
+.bcs-AvatarGroup {
+    display: flex;
+    margin: 10px 0;
+
+    .bcs-AvatarGroup-avatar {
```

Comment:
Nesting should be avoided when possible to reduce specificity. SUIT naming conventions are intended to help facilitate that by scoping styles to specific component names/namespaces.